src/App/OrganisasjonerOgTilgangerProvider.tsx

```diff
@@ -219,6 +201,25 @@ export const OrganisasjonerOgTilgangerProvider: FunctionComponent = (props) => {
                 })
         );
     }, []);
+    const userInfo = useUserInfo();
+    useEffect(() => {
```

Comment:
enkleste var å gjenbruke useState variablene via useEffect her. Må nok skrive om en del her for å kunn bruke userInfo direkte.